Library/Homebrew/utils.rb

```diff
@@ -278,7 +307,7 @@ def puts_columns(items, highlight = [])
   # TTY case: If possible, output using multiple columns.
   console_width = Tty.width
   console_width = 80 if console_width <= 0
-  max_len = items.max_by(&:length).length
+  max_len = output_length(items.max_by(&method(:output_length)))
```

Comment:
This looks a bit unwieldy. If we are running everything through `output_length` anyway, we can also move

```
plain_item_lengths = items.map(&method(:output_length))
```

here (from a few lines below) and then proceed to “compute” `max_len` from that.